setup-utils/install-mxnet-macosx.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
suggest to rename this to install mxnet-osx.sh as normally we refer the os as osx